components/grid/columns/resize.md

```diff
@@ -39,6 +39,16 @@ Autofitting specific columns preserves the current widths of all the other colum
 
 Programmatic autofitting works even if column resizing is disabled.
 
+
+### Limitations
+
+List of the known limitations of the AutoFit Columns feature:
+
+* Auto-fitting the columns on initial load of the Grid is not supported.
```

Comment:
I guess we will link to the KB here?

Reply:
Yes, now I am doing final corrections to the snippet and I will make a >note or something